### ISSUE TITLE
How to use

### ISSUE THREAD

**Issue by x2x4com** [ninemoreminutes/django-admin-object-actions#1]
Your package great, Do you have a step by step guide？

**Comment by cchurch** [comment]
http://django-admin-object-actions.readthedocs.io/en/latest/ (need to make that link more obvious).

**Comment by QwertyJack** [comment]
The doc is quite ugly :)
I just go through its [test project](https://github.com/ninemoreminutes/django-admin-object-actions/tree/master/test_project) and succeed.

**Comment by AuHau** [comment]
Yeah, you should improve the docs, because just from what is described as "set-up" it actually does not resolve in functional setup, because there is nowhere mentioned that you have to add 'display_object_actions_list'  into `list_fields` and 'display_object_actions_detail' into `fields`...

**Comment by cchurch** [comment]
Please note this project is at version 0.1 -- aka first release. Be glad there are any docs.

**Comment by atar-axis** [comment]
I really like the idea of the project and I appreciate your work there. Just without a useful documentation it is quite useless for most non-experts. I bet this project could have a lot of contributors with a little bit more documentation.

Thank you anyway, if we can help - just tell.

**Comment by cchurch** [comment]
Thanks for the interest! I mainly put this project out here to use across multiple projects of my own, but I'm glad others are also finding it useful. I haven't had too much time to do much more with it lately, but would gladly accept PRs for changes to docs or other updates/features.

**Comment by cchurch** [comment]
@AuHau I've addressed your issue in c811b10119f4de8fd9a6caf63e3a5680e14dbec4 and pushed that docs change to https://django-admin-object-actions.readthedocs.io/en/0.1.6/#usage

@atar-axis Please let me know if there are other changes to the project or docs you'd be interested in.

**Comment by atar-axis** [comment]
@cchurch: Perfect! Thank you very much - I think that was the missing link for newbies like me ;)

**Comment by AuHau** [comment]
Sweet! Thanks for the update.
